parquet/src/main/java/com/netflix/iceberg/parquet/ParquetConversions.java

```diff
@@ -33,32 +34,32 @@
   private ParquetConversions() {
   }
 
-  static <T> Literal<T> fromParquetPrimitive(Type type, Object value) {
+  static <T> ValueLiteral<T> fromParquetPrimitive(Type type, Object value) {
     if (value instanceof Boolean) {
-      return Literal.of((Boolean) value).to(type);
+      return Literals.from((Boolean) value).to(type);
```

Comment:
`Literals.from` performs duplicate `instanceof` checks. If you decide to use `from` here, then you can simplify this logic.